Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -281,6 +283,7 @@ define([
             basePath : undefined,
             incrementallyLoadTextures : false,
             upAxis : content._tileset._gltfUpAxis,
+            forwardAxis : Axis.X,
```

Comment:
This will need to get passed along through `ModelInstanceCollection` as well.

Reply:
Is there a test case, in Sandcastle maybe?

Reply:
The test tilesets all use boxes... but you can replace box.glb in [this tileset](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Specs/Data/Cesium3DTiles/Instanced/InstancedGltfExternal) with something else.

Reply:
Thanks.  Was able to reproduce and confirm the fix here too.  Looks like travis failed, I'll look into that next.